### DISCUSSION TITLE
Many erros while oppening neovim

### DISCUSSION THREAD

**Discussion by dr34mbo4t** [neovim/neovim#24774] [General]
### Problem

**### Every time I open neovim I encounter this error message containing : E5113
I use these plugins : treesitter, packer, telescope.**
### 

_```
Error detected while processing /home/anes/.config/nvim/plugin/packer_compiled.lua:
E5113: Error while calling lua chunk: /home/anes/.config/nvim/plugin/packer_compiled.lua:8: /home/anes/.config/nvim/plugin/packer_compi
led.lua: Vim(packadd):E919: Directory not found in 'packpath': "pack/*/opt/packer.nvim"
stack traceback:
        [C]: in function 'nvim_command'
        /home/anes/.config/nvim/plugin/packer_compiled.lua:8: in main chunk
Error detected while processing /home/anes/.config/nvim/after/plugin/telescope.lua:
E5113: Error while calling lua chunk: /home/anes/.config/nvim/after/plugin/telescope.lua:1: module 'telescope.builtin' not found:
        no field package.preload['telescope.builtin']
        no file './telescope/builtin.lua'
        no file '/usr/share/luajit-2.1.0-beta3/telescope/builtin.lua'
        no file '/usr/local/share/lua/5.1/telescope/builtin.lua'
        no file '/usr/local/share/lua/5.1/telescope/builtin/init.lua'
        no file '/usr/share/lua/5.1/telescope/builtin.lua'
        no file '/usr/share/lua/5.1/telescope/builtin/init.lua'
        no file './telescope/builtin.so'
        no file '/usr/local/lib/lua/5.1/telescope/builtin.so'
        no file '/usr/lib/lua/5.1/telescope/builtin.so'
        no file '/usr/local/lib/lua/5.1/loadall.so'
        no file './telescope.so'
        no file '/usr/local/lib/lua/5.1/telescope.so'
        no file '/usr/lib/lua/5.1/telescope.so'
        no file '/usr/local/lib/lua/5.1/loadall.so'
stack traceback:
        [C]: in function 'require'
        /home/anes/.config/nvim/after/plugin/telescope.lua:1: in main chunk
```_


### Steps to reproduce

nvim --clean

### Expected behavior

Opening as expected 

### Neovim version (nvim -v)

0.9.1

### Vim (not Nvim) behaves the same?

didn't try it

### Operating system/version

endeavour os 

### Terminal name/version

kitty

### $TERM environment variable

xterm-256-color

### Installation

pacman

**Comment by zeertzjq** [comment]
It's likely that you didn't install packer properly.

**Reply by dr34mbo4t** [reply]
how so??!!
